django/db/backends/base/schema.py

```diff
@@ -826,6 +853,11 @@ def alter_field(self, model, old_field, new_field, strict=False):
                 "(you cannot alter to or from M2M fields, or add or remove "
                 "through= on M2M fields)" % (old_field, new_field)
             )
+        elif old_computed_parameters != new_computed_parameters:
+            raise ValueError(
+                "Modifying ComputedFields is not supported - the field %s must "
+                "be dropped and recreated." % new_field
+            )
```

Comment:
AFAICT Postgres does not support the concept of modifying `GENERATED ALWAYS` columns, so it felt easiest to disable the ability to modify these columns altogether